scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -13,6 +13,9 @@ version. If you are using a different version, you can find each version's
 Custom Resource Definitions in the GitHub repository. For example, this
 [link contains CRDs for version 1.89.0](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/v1.89.0/crds).
 
+Note: Please review <a href="/config-connector/docs/best-practices">Best Practice for {{product_name_short}}</a> for best practices.
```

Comment:
Could you reuse the title of that page? I.e. `<a href="/config-connector/docs/best-practices">Best practices for {{product_name_short}}</a>`.

Reply:
Done